client.lua

```diff
@@ -144,15 +149,33 @@ AddEventHandler('Badssentials:RevivePlayer', function()
 	local ped = GetPlayerPed(-1);
 	if IsEntityDead(ped) then 
 		revivePed(ped);
+		TriggerEvent('chat:addMessage', {args = {Config.Prefix .. "Revived successfully!"} });

```

Comment:
Can we make such messages configurable? Probably better to have this configurable than to be plain like this.